kubernetes/apps/networking/cloudflared/app/helmrelease.yaml

```
@@ -0,0 +1,98 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: &app cloudflared
+  namespace: networking
+spec:
+  interval: 15m
+  chart:
+    spec:
+      chart: app-template
+      version: 1.4.0
+      sourceRef:
+        kind: HelmRepository
+        name: bjw-s-charts
+        namespace: flux-system
+  maxHistory: 3
+  install:
+    createNamespace: true
+    remediation:
+      retries: 3
+  upgrade:
+    cleanupOnFail: true
+    remediation:
+      retries: 3
+  uninstall:
+    keepHistory: false
+  values:
+    controller:
+      replicas: 3
+      strategy: RollingUpdate
+      annotations:
+        configmap.reloader.stakater.com/reload: &configmap cloudflared-config
+        secret.reloader.stakater.com/reload: &secret cloudflared-secret
```

This can be flattened to with the changes in the next comment.

```suggestion
      annotations:
        reloader.stakater.com/auto: "true"
```